charts/consul/values.yaml

```diff
@@ -1969,11 +1969,11 @@ connectInject:
     # @type: map
     resources:
       requests:
-        memory: "50Mi"
-        cpu: "50m"
-      limits:
         memory: "75Mi"
         cpu: "75m"
+      limits:
```

Comment:
This needed to be increased because the CNI installer was getting OOMKilled on Kind.